### PR TITLE
DOCSP-27817 fix to change facet type to facet name per DOP

### PR DESCRIPTION
**Pull request by sarahemlin**
Hey @mongoKart - 

Here's the same facet :name: fix for the Go Driver Docs repo.

Thanks!
Sarah

JIRA - <https://jira.mongodb.org/browse/DOCSP-27817>

### CODE REVIEW

File: source/fundamentals/connection.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@ Connection Guide
 .. default-domain:: mongodb
 
 .. facet::
-  :type: genre 
+  :name: genre
```
Comment (mongoKart) [inline source/fundamentals/connection.txt:10]: should these be indented another space? same for other file too.

Comment (sarahemlin) [reply source/fundamentals/connection.txt:10]: Yep!  I'll adjust.

**Review by sarahemlin — COMMENTED** [review]
@mongoKart - indentation fixed!  Ready to merge when you have a moment.

Thank you!
Sarah

**Review by mongoKart — APPROVED** [review]
lgtm

**Pull request merged**